build-zones

```diff
@@ -65,11 +51,10 @@ class Host(namedtuple('Host', ('hostname',))):
 
     @cached_property
     def ipv6(self):
-        host_type = self.ldap_attrs['type']
-
-        if host_type in HOST_TYPES_WITH_IPV6:
-            ipv6 = ipv4_to_ipv6(self.ipv4)
-            assert is_ocf_ip(ipv6), ipv6  # just in case
+        if 'ip6HostNumber' in self.ldap_attrs:
+            ipv6, = self.ldap_attrs['ip6HostNumber']
+            ipv6 = ip_address(ipv6)
+            assert is_ocf_ip(ipv6), ipv6
```

Comment:
I don't think you need the `, ipv6` here, if it's `None` or something then it won't be a valid IPv4/IPv6 object for `is_ocf_ip` to pass